CONTRIBUTING.md

```diff
@@ -1,25 +1,130 @@
+making any contribution. If you are reading this, that means you are going to be contributing to some free software,
+and that's great! Thank you!
+
+## 🗨️ Getting in touch
+
+As a Matrix based project we use chat rooms heavily for coodinating work. When getting involved with an issue or pull
+request, feel free to reach out to us in one of the project rooms. The project room for the repository you are working
+on should be visible from the README.md file.
+
+Be aware that by interacting with our Matrix rooms and/or GitHub repositories, you are agreeing to abide by the
+[Matrix.org code of conduct](https://matrix.org/legal/code-of-conduct).
+
+
+## ✍️ Filing issues
+
+We use the GitHub issue tracker for issue filing. A good issue can mean the difference between a quick fix and a long,
+painful fixing process. That's why the following guidelines exist:
+
+- If you are reporting a bug:
+  - Write a short title which neatly summaries the *problem*.
+    Do **not** write the *solution* in the issue title.
+    For example: `Cannot create a nick with | in it` is a good issue title. `Filter nicks according to RFC 2812`
+    is not a good issue title.
+  - Give a summary and as much information (along with proposed solutions) as possible in the description of the issue.
+  - Please either mention which version of the bridge you are using, or the commit hash.
+- If it's a feature:
+  - Your title should be a quick summary such as "Add ability to send encrypted files"
+  - Your description should describe the outcome, not the solution.
+    - For instance: "A function exists which can be used to send encrypted files to Matrix rooms".
+    - Not: "There should be a MatrixClient.sendEncryptedFile() so the Foobar bridge can send encrypted images"
+
+Issues will be categorised according to [Synapse's rules](https://github.com/matrix-org/synapse/issues/9460). To summarise:
+
+- Your issue will be catagorised as one of the following types by the maintainer team:
+  - **T-Defect**: Bugs, crashes, hangs, vulnerabilities, or other reported problems.
+  - **T-Enhancement**: New features, changes in functionality, performance boosts, user-facing improvements.
+  - **T-Documentation**: Improvements or additions to documentation.
+- You *may* have a severity assigned as an estimate to understand how bad the problem is and how many users are affected.
+
+The assigned labels are at the maintainers' discretion, and we will make every effort to be transparent when triaging. We do
+not, as a rule, assign priority labeling to issues.
+
+## Contributing documentation
+
+Documentation is important to us, as bridges are complex beasts and rely on good documentation for both
+administrators and users. There are a couple of things to keep in mind when when writing documentation
+for bridge projects:
+ 
+ - Our documentation is currently English, though eventually we might be able to adopt a multi-lingual documentation system.
```

Comment:
```suggestion
```

Do we have actual plans for a multi-lingual documentation system? This requires significant additional work

Reply:
Good point. Maybe our contrib file shouldn't promise what isn't  happening soon. We should at least state that it's in english though. (Or should we merge with the line about plain english....yes lets).